apps/avifenc.c

```diff
@@ -111,7 +111,7 @@ static void syntax(void)
     printf("    --duration D                      : Set all following frame durations (in timescales) to D; default 1. Can be set multiple times (before supplying each filename)\n");
     printf("    --timescale,--fps V               : Set the timescale to V. If all frames are 1 timescale in length, this is equivalent to frames per second (Default: 30)\n");
     printf("                                        If neither duration nor timescale are set, avifenc will attempt to use the framerate stored in a y4m header, if present.\n");
-    printf("    -k,--keyframe INTERVAL            : Set the forced keyframe interval (maximum frames between keyframes). Set to 0 to disable (default).\n");
+    printf("    -k,--keyframe INTERVAL            : Set the maximum keyframe interval (maximum difference between the frame indexes of any two subsequent keyframes). Set to 0 to disable (default).\n");
```

Comment:
Nit: "subsequent" doesn't sound right. Maybe "consecutive"?

("consecutive" may imply there are no frames in between, so it is also a little confusing.)

Perhaps we can go back to the previous description for now (you can find it in patchset 3 at https://aomedia-review.googlesource.com/c/libavif/+/175462) and then try to come up with a better description later.

Reply:
I updated the wording based on your discussion with @joedrago.

i think the new one looks unambiguous.